server/service/src/auth.rs

```diff
@@ -12,20 +12,11 @@ use crate::{
     token::{JWTValidationError, OmSupplyClaim, TokenService},
 };
 
-/// The enum provides some tags that can be used to tag dynamic permissions.
-/// This decouples the user permissions from the user's service capabilities.
-/// This can also be useful if there would be resources with multiple types of dynamic permissions.
-#[derive(Debug, Clone, Eq, Hash, PartialEq)]
-pub enum CapabilityTag {
-    /// Tags the list of capabilities to access documents by type
-    ContextType,
-}
-
 #[derive(Debug, Clone)]
 pub enum PermissionDSL {
     HasPermission(Permission),
     /// The permission context will be extracted and tagged with the provided tag.
```

Comment:
Is this comment still valid ?

Reply:
thanks I will update in https://github.com/openmsupply/open-msupply/pull/2047